Dockerfile

```diff
@@ -41,3 +51,26 @@ RUN mix phx.digest
 
 WORKDIR /root
 RUN mix distillery.release --verbose
+
+
+###
+# RUN THE APPLICATION
+###
+
+# The elixir image was built with debian stretch, use the node debian-stretch base
+FROM node:14.15.1-stretch
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+	libssl1.1 libsctp1 curl \
+	&& rm -rf /var/lib/apt/lists/*
+
+# Set exposed ports
+EXPOSE 4000
+ENV PORT=4000 MIX_ENV=prod TERM=xterm LANG="C.UTF-8" REPLACE_OS_VARS=true
+
+COPY --from=builder . rel/site
```

Comment:
The `.` here should be a path from the container. Base on the `build.sh` script, maybe `/root/_build/prod/rel`? The goal is to have `/root/rel/site/bin/startup` as a file that `CMD` can execute once the container is built.

The way I generally try to debug things like this is adding lines like `RUN ls -l <path>` to the Dockerfile, so I can see what files are where during the build process.

Reply:
I thought of running `RUN ls -l <path>` but for some reason it wasn't showing anything for the different paths I was giving it.
But the `/root/_build/prod/rel` path makes sense -- sorry I didn't catch that earlier.